### PR TITLE
Deconstructing Shared Nothing and Shared Disk Clustering Architectures

### LINKED ISSUE #1195: Shared Nothing and Shared Disk Clustering Architectures
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Many organizations turn to clustering as they look to increase the availability and computing capabilities of their hardware. Clustering basically works on the principle that multiple processors can tackle a problem better, faster, and more reliably than a single computer. Companies considering clustering options need to know what technologies and architectures provide the best approach to clustering. This article will discuss shared nothing and shared disk clustering architectures. 

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. Why Use Clusters?
2. Types of Clustering
3. The differences between Shared Nothing and Shared Disk Clustering Architectures in terms of scalability, availability, etc. 

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #1195

### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

File: articles/deconstructing-shared-nothing-and-shared-disk-clustering-architectures/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Shared-nothing clustering architecture is highly available. This is attributed to the fact that it can scale up quickly and cheaply. The availability of shared-disk can be limiting because the hardware used is costly compared to the shared-nothing clustering architecture.
+
+#### Data storage
+
+Shared-nothing clustering deals with data that is independently stored in separate nodes, which means that the data being stored can be partitioned and summoned when needed. Shared-disk works with information that cannot be separated.
+
+#### Load balancing
+
+[Load balancing](https://en.wikipedia.org/wiki/Load_balancing_(computing)#) is the process that a computer or processor uses to distribute tasks over specific system resources. The aim is usually to make this transmission more efficient. On that note, shared-nothing clustering architecture has a fixed load balance, whereas the shared-disk clustering architect has a dynamic loading balance, which is progressive.
+
```
Comment (lalith1403) [inline articles/deconstructing-shared-nothing-and-shared-disk-clustering-architectures/index.md:58]: Add a table summarizing the features of the 2 types of clustering. That makes this article crisp and clear.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Very well written article. A table indicating the differences would ensure the reader is left with a clean way to consume the information present.

**Comment by EKahuha** [comment]
@lalith1403, I have updated this article.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
@EKahuha  
Great overall article - seems really informative. Just found myself stumbling or having to re-read certain parts in the 
Please take keen notes on the changes I made to the intro (upper half) . 

To "save" time I did most of the edits - but otherwise I would have asked for more revisions. 

The bottom half was actually well done. 

cc: @lalith1403  please go through the edits as well for future reference. 🚀

**Pull request merged**